.github/workflows/build-docker-corsika-simtelarray-image.yml

```diff
@@ -36,15 +29,14 @@ jobs:
         type: ['simtelarray']
 
     steps:
-      - name: Download Artifact
-        uses: actions/download-artifact@v3
-        with:
-          name: corsika7.7_simtelarray.tar.gz
-          path: corsika7.7_simtelarray.tar.gz
-
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: download corsikasimtelpackage
+        run: |
+          wget --no-verbose https://syncandshare.desy.de/index.php/s/${{ secrets.CLOUD_SIMTEL }}/download
+          mv download corsika7.7_simtelarray.tar.gz
```

Comment:
not optimal to hardcode the version in the code. Couldn't we get the version already in the name of the downloaded file (question regardless of the discussion in #566)?

Reply:
Yes and no. I agree that hardcoded version numbers might create issues. But in this case it is a strong reminder on what we are doing included the problems discussed in #566. So I would prefer to leave it there until we have found a more flexible solution in obtaining corsika and sim_telarray.

Reply:
I would like to add as well that the version `7.7` mentioned here is not expected to change essentially "ever" (until we switch to CORSIKA 8 and then this would be the least of our problems).